src/audio/module_adapter/module_adapter.c

```diff
@@ -168,6 +168,9 @@ static int module_adapter_dp_queue_prepare(struct comp_dev *dev)
 	 * first, set all parameters by calling "module prepare" with pointers to
```

Comment:
"bugix" ? Also "module_adapter: " would be a better prefix. But not showstoppers, change itself looks good and git commit is understandable.